wordpress-mobile/WordPress-iOS
# Comments: merge UI feature branch into develop

Fixes #15909 

This is just to merge the Comment UI feature branch into `develop`. 

To test:
Since the code and functionality have previously been reviewed, just smoke testing the Comments should suffice.
- Go to Site > Comments.
- Verify they appear with the new fancy UI.
- Rejoice.

PR submission checklist:
